gatsby-browser.js

```diff
@@ -5,3 +5,10 @@
  */
 
 // You can delete this file if you're not using it
+
+exports.onRouteUpdate = ({ location }) => {
+  // will trigger the theme's code to re-activate
+  if (window.setup) {
+    window.setup(window.jQuery)
+  }
```

Comment:
The theme sets up JS on page load using jQuery. As Gatsby doesn't do a traditional page load jQuery doesn't trigger. This code triggers the setup instead.